### PR TITLE
deploy: Add runtimeClasses relating to the NVIDIA GPU

### PR DESCRIPTION
**Pull request by zvonkok**
Fixes: #9483

For the added configurations we need to provide runtimeClasses.

### CODE REVIEW

File: tools/packaging/kata-deploy/runtimeclasses/kata-qemu-nvidia-gpu-snp.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1
+metadata:
+    name: kata-qemu-nvidia-gpu-snp
+handler: kata-qemu-nvidia-gpu-snp
+overhead:
+    podFixed:
+        memory: "160Mi"
```
Comment (BbolroC) [inline tools/packaging/kata-deploy/runtimeclasses/kata-qemu-nvidia-gpu-snp.yaml:9]: I am seeing the values for `cpu` and `memory` for `kata-qemu-snp` are `1.0` and `2048Mi`, respectively. I would be grateful if you could explain a rationale for these reduced estimation. Thanks!

Comment (zvonkok) [reply tools/packaging/kata-deploy/runtimeclasses/kata-qemu-nvidia-gpu-snp.yaml:9]: Missed that, those were the "old" default values, let me update.

Comment (zvonkok) [reply tools/packaging/kata-deploy/runtimeclasses/kata-qemu-nvidia-gpu-snp.yaml]: Done!

**Review by fidencio — CHANGES_REQUESTED** [review]
@zvonkok, the runtime classes themselves look good, but you'd also need to:
* Add them to https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/runtimeclasses/kata-runtimeClasses.yaml
* Update https://github.com/kata-containers/kata-containers/blob/e5e0983b56534f007cf06ab39f323961b17ff642/tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml#L35
* Update https://github.com/kata-containers/kata-containers/blob/e5e0983b56534f007cf06ab39f323961b17ff642/tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml#L33

Once those are done, we can get this one merged.  Thanks!

**Comment by zvonkok** [comment]
@fidencio PTAL

**Comment by fidencio** [comment]
@zvonkok, almost there, but you need to add the 2 defnitions that you added also to the following file:
https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/runtimeclasses/kata-runtimeClasses.yaml

**Comment by fidencio** [comment]
> @zvonkok, almost there, but you need to add the 2 defnitions that you added also to the following file: https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/runtimeclasses/kata-runtimeClasses.yaml

@zvonkok, I've done the change needed and force pushed to your branch.
As soon as the tests pass I will approve and merge.

**Comment by zvonkok** [comment]
You're the best @fidencio

**Review by fidencio — APPROVED** [review]
lgtm, thanks @zvonkok!

**Pull request merged**